PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,58 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
```

Comment:
One other suggestion for these scripts: If we assume that the path will be constant the host will change, you could separate this into fields:
```
DOWNLOAD_HOST="cdn.awsstg.jumpcloud.com"
DMG_PATH="TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
REMOTE_DMG_URL="https://${DOWNLOAD_HOST}/${DMG_PATH}"
```
That might make it a little easier to update later.